examples/plot_partial_dependence_visualization_api.py

```diff
@@ -0,0 +1,110 @@
+"""
+=========================================
+Partial Dependence with Visualization API
```

Comment:
Advanced plotting with partial dependence?

Any partial dependence plot uses the visualization API by definition, right?

Reply:
Love the example btw!